crates/services/gas_price_service/src/common/fuel_core_storage_adapter/storage.rs

```diff
@@ -68,3 +70,46 @@ impl TableWithBlueprint for GasPriceMetadata {
         GasPriceColumn::State
     }
 }
+
+/// The storage for all the unrecorded blocks from gas price algorithm, used for guessing the cost
+/// for future blocks to be recorded on the DA chain
+pub struct UnrecordedBlocksTable;
+
+type BlockSizeInBytes = u64;
+
+impl Mappable for UnrecordedBlocksTable {
+    type Key = Self::OwnedKey;
+    type OwnedKey = BlockHeight;
+    type Value = Self::OwnedValue;
+    type OwnedValue = BlockSizeInBytes;
+}
+
+impl TableWithBlueprint for UnrecordedBlocksTable {
+    type Blueprint = Plain<Primitive<4>, Postcard>;
+    type Column = GasPriceColumn;
+
+    fn column() -> Self::Column {
+        GasPriceColumn::UnrecordedBlocks
+    }
+}
+
+pub struct SequenceNumberTable;
+
+/// The sequence number or bundle id of the posted blocks.
+type BundleId = u32;
```

Comment:
Why do we use both "Bundle ID" and "Sequence number" to refer to the same thing here? I'd prefer it if we can be consistent and just refer to it as "Bundle ID".

Reply:
I've changed them all to `BundleId` and `bundle_id`.

Reply:
Thanks! :pray: